File: hackerrank/30daysOfCode/python/Day 2: Operators/operators.py
import math
import sys

def solve(meal_cost, tip_percent, tax_percent):
    # initiate variable to contain total cost
    totalCost = 0

    # convert meal_cost to float
    base_cost = float(meal_cost)

    # calculate tip
    total_tip = base_cost * float(tip_percent)/100

    # calculate tax
    total_tax = base_cost * float(tax_percent)/100

    # calculate and round total cost
    totalCost = round(base_cost + total_tax + total_tip)

    print(totalCost)
    
    return totalCost

solve(sys.argv[1], sys.argv[2], sys.argv[3])

File: hackerrank/30daysOfCode/javascript/Day 2: Operators/operators.js
function solve(meal_cost, tip_percent, tax_percent) {

  let finalCost; 


  /* 
  calculate final cost by adding meal cost with tip cost and tax cost adjusted according to 
  their respecitive precentages and meal cost
  */
  finalCost = Math.floor(meal_cost + (meal_cost * tip_percent/100) + (meal_cost * tax_percent/100));

  console.log(finalCost);
}


let result = solve(12.00, 20, 30);
